### DISCUSSION TITLE
🔗 🚀 Feature Request: Taxonomy as query source in a Loop Grid (Duplicate of #21372)

### DISCUSSION THREAD

**Discussion by pixelloop** [elementor/elementor#22054] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Most page builders I use like BeaverBuilder allow you to have Taxonomy terms in the loop.

The new Elementor Loop Grid only supports post types and not taxonomies. 

### Describe the solution you'd like

Add taxonomy terms to the Loop Builder feature. 

### Describe alternatives you've considered

Not using the Elementor posts or loop builder and using WPGridBuilder instead. 

### Additional context

This would be ideal for situations where you have a custom cities taxonomy and want to have a cities overview page where you can display a list of cities (taxonomy terms) in a list and have it automatically update when new ones are added.

It would also be useful because this would then allow for search and filter/facet plugins to let the user search a long list of taxonomy terms so they can click on it and be taken to the taxonomy term archive page.

**Comment by pixelloop** [comment]
Found an existing request for this. Please merge with https://github.com/orgs/elementor/discussions/21372

**Comment by ayoubkhan558** [comment]
Duplicate of #21372

**Comment by nicholaszein** [comment]
Hello there, @pixelloop!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #21372 

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
